core/templates/components/skill-selector/skill-selector.component.ts

```diff
@@ -199,8 +201,12 @@ export class SkillSelectorComponent implements OnInit {
     });
     let filteredSkills = this.filterForMatchingSubstringPipe
       .transform(skills, searchText);
+    let currentSkill = this.skillEditorStateService.getSkill();
+
+    let activeSkillId = currentSkill ? currentSkill.getId() : null;
```

Comment:
Can it be rewritten with `let activeSkillId = currentSkill?.getId();`? 

Also use const whenever possible - here and on the line above

Reply:
Certainly, I'll update it.